### DISCUSSION TITLE
Making vanilla diving suits occupy 2 slots at once: armor and helmet

### DISCUSSION THREAD

**Discussion by Krzeszny** [FakeFishGames/Barotrauma#5300] [Ideas and suggestions]
As in the title. Diving suits aren't designed to be worn over helmets.

**Mod support:** 
Adding a parameter useful for mods that decides whether a suit occupies the head slot (disabled by default so as not to cause mods to break), as some mods have suits without helmets.

**Comment by ubertpendragon** [comment]
I'm going to assume this is a balance issue of having ballistic helmets on top of suits.
And it gets worse with security clothes

Just tested stacking **Sec clothes + Ballistic helmet + Combat suit** against a mudraptor

It took **30** mudraptor attacks to knock me down, with slight bleeding

Resistance stacking is a pretty big balance oversight. Clothes should not have built-in resistances otherwise you're begging to buy a full set of sec pants and helmets at all times on the entire crew

**Reply by CapoFantasma97** [reply]
Overalls of different roles have different stats. Engineer's hazmat suit protects from radiation, burns and lacerations caused by work failures. All stuff that one would want to have when working around maintenance and reactor handling.

Protection from enemy damage isn't always the best type of resistance you might want to look after. Especially if you're not the one going outside the submarine to explore dungeons, but the one sitting in the sub, repairing it.

**Reply by Krzeszny** [reply]
That's untrue! Most roles don't have protective stats or have the wrong protective stats. Captains have no stats. Pharmacologists don't have stats even though they use reskinned first responder clothes. Mechanics' overalls don't protect from lacerations, only from burns (burns are an electrical hazard.)

**Comment by Likvidus** [comment]
Maybe do something like a situation where you are trying to put on a diving suit while wearing body armor, but with a helmet. I think It will be much easier.

**Reply by Krzeszny** [reply]
How? Diving suits replace body armor because they use the same slot. The easiest solution would be to make them take both the body armor and the helmet slot, which some mods do so it's easy.

**Comment by Ek-Videogames** [comment]
Some basic pointers:
- We'll need a container to store helmets such as breaking diving locker into 2 parts.
- Pressure protection combines in weird ways and would need a workaround 
like how the old BTE wetsuit used to work & current mods work now
- 'Face' items such as health scanner don't make sense being a shared head slot to begin with.

**EDIT:**
- Bot AI changes ?
